### DISCUSSION TITLE
Discord Webhook / Notification Agent

### DISCUSSION THREAD

**Discussion by Lucy-2077** [filebrowser/filebrowser#1199] [Q&A]
Hi , my idea is for add the possibility for every action on filebrowser to make a message via webhook for exemple
The possibility to make notification agent with discord , mail...
Every action can be trigger for send a notification with notification agent

**Comment by o1egl** [comment]
You can write a script and attach it to any hook in `Settings -> Global Settings -> Command runner`

**Comment by Lucy-2077** [comment]
Yes but we can't make a action when a user download a file exemple
